docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -0,0 +1,150 @@
+:page_id: stack-monitoring
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Stack Monitoring
+
+You can enable link:https://www.elastic.co/guide/en/elasticsearch/reference/current/monitor-elasticsearch-cluster.html[Stack Monitoring]
+on your Elasticsearch and Kibana to collect and ship their metrics and logs.
+
+You just have to reference the separate monitoring Elasticsearch cluster in the `spec.monitoring` section.
+
+[source,yaml,subs="attributes,callouts"]
+----
+cat $$<<$$EOF | kubectl apply -f -
+apiVersion: elasticsearch.k8s.elastic.co/{eck_crd_version}
+kind: Elasticsearch
+metadata:
+  name: monitored-sample
+  namespace: production
+spec:
+  version: {version}
+  monitoring:
+    metrics:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+    logs:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+  nodeSets:
+  - name: default
+    count: 1
+    config:
+      node.store.allow_mmap: false
+---
+apiVersion: kibana.k8s.elastic.co/{eck_crd_version}
+kind: Kibana
+metadata:
+  name: monitored-sample
+  namespace: production
+spec:
+  version: {version}
+  elasticsearchRef:
+    name: monitored-sample
+    namespace: production
+  monitoring:
+    metrics:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+    logs:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+  count: 1
+EOF
+----
+
+The use of `namespace` is optional if the Elasticsearch cluster and the Kibana are running in the same namespace.
+
+The monitoring cluster must be managed by ECK in the same Kubernetes cluster as the monitored one.
+
+You can send metrics and logs to two different Elasticsearch monitoring clusters.
+
+You can enable Stack Monitoring only on Elasticsearch or only on Kibana. In the latter case, Kibana will not be available on the Stack Monitoring Kibana page (see link:https://www.elastic.co/guide/en/kibana/current/monitoring-data.html#monitoring-data(View monitoring data in Kibana)).
+
+== How it works
+
+In the background, Metricbeat and Filebeat are deployed as sidecar containers in the same Pod than Elasticsearch and Kibana.
+
+Metricbeat is used to collect monitoring metrics and Filebeat to monitor the Elasticsearch log files, collect log events. The two beats are configured to ship data directly to the monitoring cluster(s) using dedicated Elastic users managed by ECK.
+
+Elasticsearch and Kibana containers are configured to share the
```

Comment:
Incomplete sentence?

Reply:
Whoooops. I was about to explain the logs volume sharing for Filebeat. But I'm not sure how far to go in explaining how it works. It seems to me that mentioning the sidecars is the most important part and the rest is secondary.